framework/aster-frame/src/mm/mod.rs

```diff
@@ -27,7 +27,7 @@ use spin::Once;
 pub use self::{
     dma::{Daddr, DmaCoherent, DmaDirection, DmaStream, DmaStreamSlice, HasDaddr},
     frame::{options::FrameAllocOptions, Frame, FrameVec, FrameVecIter, Segment},
-    io::{VmIo, VmReader, VmWriter},
+    io::{KernelSpace, UserSpace, VmIo, VmReader, VmWriter},
```

Comment:
Maybe it's time to discuss the name `VmIo` as an generalization. Oftentimes `VmIo` is not used for memory mapped I/O but just for copying or setting values. And the implementation also just operates regardless whether is physically backed mapping or devices.

`VmRw` seems a better alternative. As memory mapped I/O is just read or write to virtual memory addresses.